lib/exercism/five_a_day_count.rb

```diff
@@ -0,0 +1,42 @@
+class FiveADayCount < ActiveRecord::Base
+  belongs_to :user
+  # has_and_belongs_to_many :user_exercise
+  #
+  # scope :today, lambda { where(day: Date.today) }
+
+  def five_suggestions_count
+    sql = "select total
+           from five_a_day_counts
+           where user_id = #{user.id}"
```

Comment:
I took the most recent version of the branch you gave me, were there changes to it afterwards?